### PR TITLE
Do not send E-Mail notifications for restarted jobs

### PR DESCRIPTION
**Pull request by alvarocarvajald**
Proposal: avoid sending E-Mail notifications for jobs that were automatically restarted (for example, via **RETRY** setting) by checking if the job has been cloned.

### CODE REVIEW

**Comment by perlpunk** [comment]
I pushed a fix for the test

**Comment by alvarocarvajald** [comment]
> I pushed a fix for the test

Oh, thanks @perlpunk

**Review by okurz — CHANGES_REQUESTED** [review]
This goes in the right direction but IMHO we shouldn't even have the complete script called on jobs that are already retried. Maybe a regression in openQA?

**Review by okurz — APPROVED** [review]
Thought about it and it looks like your approach is actually completely fine. We might want to consider moving the logic that early-aborts on retry some lines above so that we abort `handle_unknown` earlier but we should think about that further. So good as is, thank you!

File: _common
Comment (Martchus) [inline _common:135]:
```suggestion
        clone_id=$(echo "$job_data" | runjq -r '.job.clone_id')
```

**Review by Martchus — APPROVED** [review]

File: _common
Comment (alvarocarvajald) [reply _common:135]: Did not notice the double double quotes there. It works though:
```
$ job_data='{"job": {"name": "foo", "result": "failed"}}'
$ A="$(echo "$job_data" | jq -r '.job.clone_id')"
$ echo $A
null
$
```
However this got merged right after your approval. Should I submit a new PR to get rid of the outermost double quotes, or leave as is?

Comment (Martchus) [reply _common:135]: You can leave it as-is. I've just mentioned it because quotes are not necessary when assigning variables like this. (They to no harm of course.)

**Pull request merged**
